lib/twitter/accounts.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Twitter.Accounts do
+  @moduledoc """
+  The Accounts context.
+  """
+  alias Twitter.Repo
+  alias Twitter.Accounts.User
+
+  def create_user(attrs \\ %{}) do
```

Comment:
can try adding some documentation for public `def` functions, using [typespecs](https://hexdocs.pm/elixir/1.12/typespecs.html) and [docs](https://hexdocs.pm/elixir/1.12/writing-documentation.html)

For production app we usually document almost all the public functions so that it is easier for someone else to navigate the codebase